src/licensedcode/data/rules/not-a-license_431.RULE

```diff
@@ -0,0 +1 @@
+label	= "GPL3"
```

Comment:
Same comment as for the `label = gpl2` rule

Reply:
Yes, will add a line before.